controllers/cloudinit/utils.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudinit
+
+import (
+	"fmt"
+	"strconv"
+
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+func createInstallArgs(confinement string, kubernetesVersion *version.Version) string {
+	kubernetesMajorVersion := strconv.FormatUint(uint64(kubernetesVersion.Major()), 10)
+	kubernetesMinorVersion := strconv.FormatUint(uint64(kubernetesVersion.Minor()), 10)
+	installChannel := fmt.Sprintf("%s.%s", kubernetesMajorVersion, kubernetesMinorVersion)
+	var installArgs string
+	if confinement == "strict" {
+		installArgs = fmt.Sprintf("%s-strict", installChannel)
+	} else {
+		installArgs = fmt.Sprintf("%s --classic", installChannel)
```

Comment:
Maybe include the `--channel` part here, otherwise the `--channel $channel` is split over two files that are too far away from each other.

Reply:
Makes sense. Done.